ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/dot1x.schema.yml

```diff
@@ -147,3 +147,21 @@ keys:
             min: 5
             max: 65535
             description: Interval period in seconds.
+      captive_portal:
+        type: dict
+        description: Web authentication feature that authenticates a supplicant through a web page, referred to as a captive portal.
+        keys:
+          url:
+            type: str
```

Comment:
I think we should mention in description that url takes only `http` or `https` links as we have done for `monitor_telemetry_influx.schema`

Reply:
fixed